librz/include/rz_analysis.h

```diff
@@ -535,6 +537,7 @@ typedef struct rz_analysis_t {
 	RzAnalysisDebugInfo *debug_info; ///< store all debug info parsed from DWARF, etc..
 	ut64 cmpval; ///< last compare value for jump table.
 	ut64 lea_jmptbl_ip; ///< jump table x86 lea ip
+	RzVmlinuxConfig* vmlinux_config;
```

Comment:
I am not particularly happy about adding the vmlinux config directly in the RzAnalysis; honestly, it looks out of place. @wargio @ret2libc @Rot127 @thestr4ng3r any ideas?

Reply:
The proper way would be to have it in it's own plugin IMHO. Because the logic is very much contained. But it is good how it is currently. Wouldn't know a place where the config fits else.
Added it https://github.com/rizinorg/rizin/issues/4334 as code which should be move to it's own plugin after refactoring. Please remove it if you disagree.

Reply:
i wonder if this should be a bin+io plugin with, like we have for `dmp`